libtenzir/include/tenzir/multi_series_builder.hpp

```diff
@@ -343,6 +343,9 @@ class multi_series_builder {
     // created by `data_unparsed` calls, unless those fields are present in
     // the used schema.
     bool raw = false;
+    // How to handle duplicate keys in non-merging mode.
+    detail::data_builder::duplicate_keys duplicate_keys
+      = detail::data_builder::duplicate_keys::merge_structural;
```

Comment:
The data builder has the default set to `overwrite`. How do the two defaults interact?